cloud/amazon/ec2_eni.py

```diff
@@ -341,6 +355,37 @@ def get_sec_group_list(groups):
 
     return remote_security_groups
 
+def make_securitygroup_list(connection, module, group_list,subnet_id,eni_id):
```

Comment:
See https://github.com/ansible/ansible-modules-extras/pull/953.

This is a good idea but it should be shared code.  I'm sure @bretmartin wouldn't mind if you wanted to take on the task of implementing this code centrally.